### PR TITLE
Adding content to Win forensics and other tools; Adding sections for OSX forensics,Sandboxing/reversing and Books

### PR DESCRIPTION
**Pull request by mikesxrs**
Added misc information I have found useful

-Highlighter from Fire/Mandiant
-RegRipper Registry tool for win forensics
-OSX Evidence Section
-Sandboxing/reversing tools (both local and online)
-Book Section with Amazon link (feel free to change)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +68,17 @@ A curated list of tools and resources for security incident response, aimed to h
...
+* [RegRipper](https://code.google.com/p/regripper/wiki/RegRipper) - Regripper is an open source tool, written in Perl, for extracting/parsing information (keys, values, data) from the Registry and presenting it for analysis.
 
+### OSX Evidence Collection
+* [OSX Auditor](https://github.com/jipegit/OSXAuditor) - OSX Auditor is a free Mac OS X computer forensics tool
+
+### Sandboxing/reversing tools
+* [Cuckoo](https://github.com/cuckoobox) - Open Source Highly configurable sandboxing tool 
+* [Mastiff](https://github.com/KoreLogicSecurity/mastiff) - MASTIFF is a static analysis framework that automates the process of extracting key characteristics from a number of different file formats.
+* [Viper](https://github.com/viper-framework/viper) - Viper is a python based binary analysis and management framework, that works well with Cuckoo and YARA.
+* [Virustotal](https://irustotal.com) - Virustotal, a subsidiary of Google, is a free online service that analyzes files and URLs enabling the identification of viruses, worms, trojans and other kinds of malicious content detected by antivirus engines and website scanners
```
Comment (meirwah) [inline README.md]: still broken link : 
missing a "V" in the link here : https://virustotal.com

**Comment by meirwah** [comment]
Heyyy @mikesxrs why did you close this?
just need to fix missing 'v' and I can merge this PR...

**Comment by mikesxrs** [comment]
@meirwah lets try this again

**Comment by meirwah** [comment]
:+1:

**Comment by hslatman** [comment]
Nice category additions!

**Pull request merged**